android/java/res/drawable/ic_logo_zebpay.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="16dp"
+    android:height="12dp"
+    android:viewportWidth="1697.4"
+    android:viewportHeight="1251.1">
+    <path
+        android:fillColor="#0273F2"
```

Comment:
this file only color is different from file : android/java/res/drawable/ic_logo_zebpay_white.xml
android:fillColor="#0273F2",  android:fillColor="#FFFFFF"
hope we can just override color in color.xml instead of two file

Reply:
good call. it's not used for light/night mode. we need to use different icons for different use case. i will use tint in this case.